app/Models/Scopes/Pinnable.php

```diff
@@ -5,12 +5,12 @@
 use Illuminate\Database\Eloquent\Builder;
 
 /**
- * Trait Pinned
+ * Trait Pinnable
  * @package App\Models\Scopes
```

Comment:
We can add a `@property bool $is_pinned` here to make help with the IDE auto-complete code, and now need it on the models themselves :)